environment.yml

```diff
@@ -0,0 +1,30 @@
+name: aas229-workshop
+
+channels:
+  - http://ssb.stsci.edu/astroconda
+
+dependencies:
```

Comment:
are you going to demo stginga? Since you are also demoing specviz and glue and ginga, please make sure all the versions are correct for those packages.